pages/data_life_cycle/preserving.md

```diff
@@ -69,6 +69,13 @@ When preparing data for preservation several requirements need to be fulfilled.
 - Ensure that a consistent naming scheme is used.
 - Use standard, open source, file formats instead of proprietary ones.
 
+<!-- ERIM specific info (begin)-->
+
+At EUR we recommend researchers to use [EUR Yoda Vault](https://my.eur.nl/en/eur-employee/data-archiving-eur-yoda-vault) in order for reseachers to archive and preserve their data for verification purposes. 
```

Comment:
I'm just suggesting this to change to the public URL 👍 

```suggestion
At EUR we recommend researchers to use [EUR Yoda Vault](https://eur.nl/en/data-archiving-eur-yoda-vault) in order for reseachers to archive and preserve their data for verification purposes. 
```

Reply:
Yeah way better to use the public links! :)